source/Scoreboard.cpp

```diff
@@ -0,0 +1,70 @@
+#include "../include/Scoreboard.hpp"
```

Comment:
Please just use `#include "ScoreBoard.hpp"`. If it is in the `include` folder the makefile should find it.